dev-libs/libzip/libzip-1.2.0.recipe

```diff
@@ -73,14 +73,14 @@ INSTALL()
 	prepareInstalledDevelLibs libzip
 	fixPkgconfig
 
-	# Move zipconf.h from lib/libzip/include/ to develop/headers/libzip/
-	mkdir -p $includeDir/libzip
-	mv $libDir/libzip/include/zipconf.h $includeDir/libzip
-	rmdir $libDir/libzip/include $libDir/libzip
+	# Move zipconf.h from lib/libzip/include/ to develop/headers/
+	mkdir -p $includeDir
+	cp $libDir/libzip/include/zipconf.h $includeDir
+	rm -R $libDir/libzip
 
 	# Adjust libincludedir to make it point to where zipconf.h was moved.
-	sed -i -e "s,^\(libincludedir=\)$libDir"'\(/libzip\)/include$,\1${prefix}/'"${relativeIncludeDir}\2," \
-		$developLibDir/pkgconfig/libzip.pc
+	#sed -i -e "s,^\(libincludedir=\)$includeDir"'\(/libzip\)/include$,\1${prefix}/'"${relativeIncludeDir}\2," \
+	#	$developLibDir/pkgconfig/libzip.pc
```

Comment:
from the libzip.pc file, so it looks like those can be removed?

  prefix=/packages/libzip-1.2.0-2/.self
  exec_prefix=${prefix}
  libdir=/packages/libzip-1.2.0-2/.self/lib
  includedir=/packages/libzip-1.2.0-2/.self/develop/headers
  libincludedir=/packages/libzip-1.2.0-2/.self/develop/headers